app/jobs/reminder_deadline_job.rb

```diff
@@ -1,13 +1,9 @@
 class ReminderDeadlineJob < ApplicationJob
```

Comment:
Helpful thing to know is this gets called through a scheduled job on heroku. We run lib/tasks/initiate_reminder_deadline_job.rake every day through that approach.

Reply:
Good point. Adding!